Cargo.toml

```diff
@@ -18,8 +18,15 @@ tokio = { version = "1", features = [], default-features=false, optional = true
 
 [features]
 # todo: remove before merge
-default = ["tokio_io"]
+default = ["tokio_io", "group_size_16k"]
 tokio_io = ["tokio", "futures"]
+group_size_1k = [] # compatible with bao
+group_size_2k = []
+group_size_4k = []
+group_size_8k = []
+group_size_16k = []
+group_size_32k = []
+group_size_64k = []
```

Comment:
can we add a 256k option as well?

Reply:
actually, I'd love to see 128, 256, 512, and 1024k.

Reply:
Sure.